src/metric_config.cpp

```diff
@@ -44,6 +45,13 @@ const std::string METRIC_NAME_CURRENT_REQUESTS = "ovms_current_requests";
 const std::string METRIC_NAME_REQUEST_TIME = "ovms_request_time_us";
 const std::string METRIC_NAME_WAIT_FOR_INFER_REQ_TIME = "ovms_wait_for_infer_req_time_us";
 
+// MediaPipe
+const std::string METRIC_NAME_CURRENT_GRAPHS = "ovms_current_graphs";
+const std::string METRIC_NAME_RESPONSES = "ovms_responses";
+
+const std::string METRIC_NAME_REQUESTS_ACCEPTED = "ovms_requests_accepted";
+const std::string METRIC_NAME_REQUESTS_REJECTED = "ovms_requests_rejected";
```

Comment:
There is missing metric for fail in graph execution.

Reply:
To be done in separate PR, added to PR description